roles/maintenance_32_apache/tasks/main.yml

```
@@ -0,0 +1,278 @@
+
+- <<: *task
+  vars:
+    taskid: 32-010
+    name: "Apache: Are there files in the apache log dir that are not rotated? | Find large files"
+    # 4096 blocks equals 16M (assuming 4k logical blocksize, which is to be expected nowadays)
+    apache_large_files: "{{ apache_large_logfiles_stat.results | map(attribute='stat') | selectattr('blocks', 'gt', 4096) | map(attribute='path') | list }}"
+  ansible.builtin.debug:
+    var: "apache_large_files"
+  changed_when: "apache_large_files | length > 0"
+
+# Check the used config file validity
+- <<: *task
+  vars:
+    taskid: 32-012
+    name: "Apache: Check apache configuration validity"
+  ansible.builtin.command: "{{ apache_binary_path }} -t"
+  register: apache_config_test
+  changed_when: "apache_config_test.rc != 0"
+
+# Test if cert is valid in one month
+- <<: *task
+  vars:
+    taskid: 32-014
+    name: Test whether that certificate is valid in a month
+  ansible.builtin.slurp:
+    src: "{{ item }}"
+  register: apache_cert_files_content
+  loop: "{{ apache_cert_files }}"
+  changed_when: false
+
+- <<: *task
+  vars:
+    taskid: 32-014
+    name: Test whether that certificate is valid in a month
+  community.crypto.x509_certificate_info:
+    content: "{{ item }}"
+    valid_at:
+      one_month_later: "+1m"
+  register: apache_cert_info
+  loop: >-
+    {%- set cert_content = [] -%}
+    {%- for result in apache_cert_files_content.results -%}
+    {%- set cert_content = cert_content + ([(result.content | b64decode | community.crypto.split_pem)] | flatten) -%}
+    {%- endfor -%}
+    {{- cert_content -}}
+  changed_when: false
+  ignore_errors: true
+  delegate_to: localhost
+
+- <<: *task
+  vars:
+    taskid: 32-014
+    name: Test whether that certificate is valid in a month
+  ansible.builtin.debug:
+    msg: "{{ item.not_after }}: {{ item.subject }}"
+  loop: "{{ apache_cert_info.results }}"
+  when: not (item.valid_at.one_month_later | default(true))
+  changed_when: true
+
+# Not printed out by default if you want to see for debugging, change changed_when to yes
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Print apache run details"
+  ansible.builtin.debug:
+    msg:
+      - "HTTPD root directory: {{ apache_root_directory if apache_root_directory is defined else 'Not defined' }}"
+      - "Config file name: {{ apache_config_file_name if apache_config_file_name is defined else 'Not defined' }}"
+      - "logfile names: {{ apache_log_files if apache_log_files is defined else 'Not defined' }}"
+      - "Is SSL active: {{ apache_is_ssl_active if apache_is_ssl_active is defined else 'Not defined' }}"
+      - "Included apache config paths: {{ apache_included_config_paths if apache_included_config_paths is defined else 'Not defined' }}"
+      - "SSL cert paths: {{ apache_cert_files if apache_cert_files is defined else 'Not defined' }}"
+  changed_when: false
+
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Print apache logfiles"
```

```suggestion
    taskid: 32-011
    name: "Apache: Are there any problematic messages in the error logs?"
```

Please also move this task up between 32-010 and 32-012